src/decisionengine/framework/dataspace/tests/test_datablock.py

```diff
@@ -205,6 +205,15 @@ def test_DataBlock_get_taskmanagers(self):
         tms = self.dataspace.get_taskmanagers()
         self.assertEqual(taskmanager["taskmanager_id"], tms[0]["taskmanager_id"])
 
+    def test_DataBlock_is_expired(self):
+        self.datablock.is_expired()
+
+    def test_DataBlock_is_expired_with_key(self):
+        dataproduct = self.data["dataproduct"][0]
+        self.datablock.is_expired(key=dataproduct["key"])
+
+    def test_DataBlock_mark_expired(self):
+        self.datablock.mark_expired(1)
```

Comment:
These tests exercise the code, but they don't test behavior.  Because this functionality actually isn't implemented, we should at least indicate that in the tests through assertions.  For example, a reasonable return type for `is_expired(...)` would be `bool`, but because the feature is unimplemented, it returns `None`.  Could you thus use assertions accordingly (e.g.):

```python
def test_DataBlock_is_expired(self):
    assert self.datablock.is_expired() is None  # eventually will be True or False
```
and so on for the others.

Reply:
I believe I'e updated the code to match expectations (and added a blob of comment for long term thoughts.)